auto_examples/index.rst

```diff
@@ -30,7 +30,7 @@ of using the Flux Python API.
 .. only:: html
 
   .. image:: /auto_examples/images/thumb/sphx_glr_example_job_submit_api_thumb.png
-    :alt: Introductory example - Job Submit API
+    :alt:
```

Comment:
was this removal accidental?  seems out of place?

Reply:
Yeah I thought so too - let me poke around. What it probably means is that someone merged without updating the examples (and then missed this). It doesn't make sense it wouldn't be updated.

Reply:
It's just alt text for the image - I'm OK to remove it if we can't figure it out. It is a mystery though!

Reply:
ok approved!

Reply:
Also - I should have stated this directly - the auto_examples directory is generated by the `flux start make html` as part of sphinx gallery, it's not generated by a human. The reason the other PRs are failing (aside from the sys import bug) is because it's finding this difference between what is rendered and what should be.